tasks/definevscvm.yml

```diff
@@ -0,0 +1,37 @@
+# List running VM
+- name: Get a list of VMs
+  virt: command=list_vms
+  register: virt_vms
+  delegate_to: "{{ hypervisor }}"
+
+# Destroy if running VSC Guest VM
+- name: Destroy VSC Guest VM if still running
+  virt: name={{ inventory_hostname }} state=destroyed
+  delegate_to: "{{ hypervisor }}"
+  when: inventory_hostname in virt_vms.list_vms  
+
+# Undefine any previously existing VSC Guest VM
+- name: Undefine VSC Guest VM if already provisioned
+  virt: name={{ inventory_hostname }} command=undefine
+  delegate_to: "{{ hypervisor }}"
+  when: inventory_hostname in virt_vms.list_vms
+
+- name: Copy over VSC qcow2 image
```

Comment:
It seems like another copying procedure. The first one is still in place in `main.yml`